### ISSUE TITLE
Logo de superestrella

### ISSUE THREAD

**Issue by Vincelot-Dev** [RodrigoToroIcarte/IIC2113-2023-1#128]
Hola, tengo una duda con respecto a la regla número 4 para definir si un mazo es inválido
![image](https://github.com/RodrigoToroIcarte/IIC2113-2023-1/assets/80913968/2423c7fd-110a-420f-8945-da6d55ff11b4)
Esta regla también debe ser aplicada en la entrega 1?, y de ser así, como se determina eso?

**Comment by Vincelot-Dev** [comment]
Ahi vi que sale en los subtipos, no me habia quedado claro con el enunciado

**Comment by RodrigoToroIcarte** [comment]
Sí, ver https://github.com/RodrigoToroIcarte/IIC2113-2023-1/issues/119 para más información.
